src/util/upnp_quirks.h

```diff
@@ -31,10 +31,15 @@
 #include <pugixml.hpp>
 namespace fs = std::filesystem;
 
+class ActionRequest;
```

Comment:
These should go with the other forward declarations below

Reply:
Resolved in https://github.com/gerbera/gerbera/pull/1288/commits/cb16aa7074d22b899544f986df6b1d0821b2e66f